packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.test.js

```diff
@@ -29,6 +29,11 @@ describe('<SpeedDialAction />', () => {
     wrapper.unmount();
   });
 
+  it('initializes its state from props', () => {
+    const wrapper = shallow(<SpeedDialAction {...defaultProps} open tooltipOpen />);
+    assert.strictEqual(wrapper.state('tooltipOpen'), true);
```

Comment:
`wrapper.state('x')` vs `wrapper.state().x` what do you think?

Reply:
@oliviertassinari  I think the right answer is `wrapper.state().x` ;)